intellij-ext/modules/core/src/main/kotlin/com/metalbear/mirrord/MirrordNpmExecutionListener.kt

```diff
@@ -0,0 +1,180 @@
+package com.metalbear.mirrord
+
+import com.intellij.execution.ExecutionListener
+import com.intellij.execution.configuration.EnvironmentVariablesData
+import com.intellij.execution.configurations.GeneralCommandLine
+import com.intellij.execution.process.ProcessHandler
+import com.intellij.execution.runners.ExecutionEnvironment
+import com.intellij.execution.target.createEnvironmentRequest
+import com.intellij.execution.wsl.WSLDistribution
+import com.intellij.execution.wsl.target.WslTargetEnvironmentRequest
+import com.intellij.notification.*
+import com.intellij.openapi.util.SystemInfo
+import java.util.concurrent.TimeUnit
+
+
+class MirrordNpmExecutionListener : ExecutionListener {
+
+	companion object {
+		var mirrordEnv: LinkedHashMap<String, String> = LinkedHashMap()
```

Comment:
What happens if you run 2 npm services at the same time? I assume everything breaks and hell is brought upon us..
I think you should use `getExecutionId` then have it per execution id.